Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -406,14 +522,24 @@ def ensure_chrome_running():  # pragma: no cover
             # There's no Chrome listening on that port, Start a new Chrome there
             demisto.debug(f"No Chrome found on port {chrome_port}")
             demisto.debug(f'Initializing a new Chrome session on port {chrome_port}')
```

Comment:
You're not initializing a new chrome here anymore...
```suggestion
            demisto.debug(f"No Chrome found on port {chrome_port}, using it.")
```